src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,79 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+public class Parser {
+    private boolean bye;
+
+    public Parser() {
+        this.bye = false;
+    }
+
+    public boolean isBye() {
```

Comment:
Could there have been a better name for this method? `isBye` is might not make sense to someone reading your code.